### ISSUE TITLE
sys-usb with usb-keyboard enabled causes lag on frequent user sessions changes.

### ISSUE THREAD

**Issue by ZYousef** [QubesOS/qubes-issues#7953]
[How to file a helpful issue](https://www.qubes-os.org/doc/issue-tracking/)

### Qubes OS release
QR4.1

### Brief summary

Login in & out of Xfce session frequently causes the system to lag when having usb-keyboard setup enabled for sys-usb.

### Steps to reproduce

1. Login into new Xfce session.
2. Logout of the session. 
Repat 1-2 for a few times.

Now You will notice some performance degradation in the guivm, lagging stuttering, and unresponsiveness.

### Expected behavior
The system should properly function even after starting new sessions.

### Actual behavior
The system becomes laggy, unresponsive and sometimes unusable to do sys-usb crash.

**Comment by marmarek** [comment]
Do you see increased CPU usage in sys-usb? Or some frequent kernel messages there?

**Comment by ZYousef** [comment]
Yes exactly, CPU usage in sys-usb increased above 50% and I get a kernel message:
hid-generic control queue full.

I found two workarounds: usbhid-dump or reconnecting the keyboard.
I set a service that listens for the kernel msg and dumps his driver for now

**Comment by marmarek** [comment]
I can trigger the issue by pressing Caps Lock 4 times, then I get _a lot of_ `control queue full` messages in related USB qube. I'm not sure if that's specific to my specific keyboard model

**Comment by ZYousef** [comment]
> I can trigger the issue by pressing Caps Lock 4 times, then I get _a lot of_ `control queue full` messages in related USB qube. I'm not sure if that's specific to my specific keyboard model

I tried pressing Caps Lock it seems to work if the input is focused on guivm, while I'm focused on usbvm terminal It doesn't cause in issues. When the mouse becomes laggy I found some logs in usbvm showing that it's coming from a function called hidinput_led_worker.

**Comment by ZYousef** [comment]
I found out that when the warning is written to the kernel buffer it causes the lag, I mitigated this by preventing the warning message from being written to the buffer using `sudo dmesg -n err`. That does not fix the problem but it allows the keyboard and mouse to function normally when it occurs.
To make this change permanent I set the command inside `/rw/config/rc.local`

**Comment by marmarek** [comment]
I suspect this is something that https://github.com/QubesOS/qubes-app-linux-input-proxy/ is doing wrong, but I have no idea what.
